draft-chen-cfrg-vdaf-pine.md

```diff
@@ -307,8 +307,33 @@ We define a function `PineValid.encode_gradient(self, measurement: list[float])
 
 ### Running the Wraparound Tests {#run-wr-check}
 
-> XXX Give an overview of how the wraparound test results are generated from
-> the XOF seed.
+Given the encoded gradient from {{encode-gradient}} and the XOF to generate the
+random vectors, the Client needs to run wraparound check for `num_wr_checks`
+times, and is required to pass at least `num_wr_successes` of them. Each
+wraparound check works as the following:
+
+The Client generates a random vector, which has the same dimension as the
+gradient `dimension`. Each entry of the random vector is a field element of `1`
```

Comment:
nit (grammar): It sounds like you're using the variable `dimension` to denote the gradient.